reduce delete cron exception log

anandjakhaniya opened the pull request:
**Jira Ticket:** https://vertaai.atlassian.net/browse/VR-4951

anandjakhaniya committed:
reduce delete cron exception log

anandjakhaniya committed:
revert code and add it in other PR

ravishetye commented on backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java:
```diff
@@ -74,8 +76,16 @@ public void run() {
     } catch (Exception ex) {
-      ex.printStackTrace();
-      LOGGER.error("DeleteEntitiesCron Exception: ", ex);
+      if (ex instanceof StatusRuntimeException) {
```
we need to refactor the error handling since this way if one project throws exception the entire flow does not proceed.

ravishetye commented:
marking the flow in my head for projects, we can adopt it for other by making corresponding changes

1. try catch log around deleteRoleBindingsForProjects(projectEntities);
1. try catch log around session.delete(projectEntity);

anandjakhaniya committed:
Added try catch log around deleteRoleBindingsFor(*)
Added try catch log around ransaction.commit() after session.delete(*entities);

anandjakhaniya committed:
Merge branch 'master' into aj/reduce-cron-log

# Conflicts:
#	backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java
